app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -127,6 +128,27 @@ function listenForNotificationsBellClick() {
   }, 180);
 }
 
+function initFilter() {
```

Comment:
We don't have a good habit of adding documentation around new JS code, but I would definitely encourage adding docs anytime we add new code :)

I believe we're using [JSDoc](https://jsdoc.app/), and there's [a good example of how to use it](https://github.com/thepracticaldev/dev.to/blob/d461d835405e023630bf694037fccef7efaea8df/app/javascript/src/utils/debounceAction.js#L3-L16) in some of our preexisting JS functions!

Reply:
Gotcha! I'm copying the other ways we're using JS as a filter — the code I borrowed is from the "chronofilter" on the home feed where we let users filter their feed based on Feed / Week / Month / Year/ Infinity.